forest/cli/src/cli/fetch_params_cmd.rs

```diff
@@ -66,13 +66,37 @@ mod tests {
 
     #[test]
     fn ram_str_conversions() {
-        assert_eq!(ram_to_int("2048").unwrap(), SectorSize::_2KiB);
-        assert_eq!(ram_to_int("2048B").unwrap(), SectorSize::_2KiB);
-        assert_eq!(ram_to_int("2kib").unwrap(), SectorSize::_2KiB);
-        assert_eq!(ram_to_int("8Mib").unwrap(), SectorSize::_8MiB);
-        assert_eq!(ram_to_int("512MiB").unwrap(), SectorSize::_512MiB);
-        assert_eq!(ram_to_int("32Gi").unwrap(), SectorSize::_32GiB);
-        assert_eq!(ram_to_int("32GiB").unwrap(), SectorSize::_32GiB);
-        assert_eq!(ram_to_int("64Gib").unwrap(), SectorSize::_64GiB);
+        assert_eq!(
+            ram_to_int("2048").unwrap(),
+            <SectorSize as Inner>::FVM::_2KiB.into()
+        );
```

Comment:
You can use a local type alias here:

```suggestion
        type SectorSize = <super::SectorSize as super::Inner>::FVM;
        assert_eq!(ram_to_int("2048").unwrap(), SectorSize::_2KiB.into());
```

Reply:
done

Reply:
It doesn't look done to me. Maybe you forgot to push your changes? In the future, make sure you push all your changes before replying that something is done.

Reply:
sorry, there was a delay in pushing my changes